pyrato/__init__.py

```diff
@@ -1,48 +1,26 @@
 # -*- coding: utf-8 -*-
 
 __author__ = \
-    """Marco Berzborn - Institute for Hearing Technology and Acoustics"""
-__email__ = 'marco.berzborn@akustik.rwth-aachen.de'
+    """The pyfar developers"""
+__email__ = 'info@pyfar.org'
 __version__ = '0.3.2'
 
-from .rap import (
-    reverberation_time_linear_regression
-)
-from .roomacoustics import (
-    reverberation_time_energy_decay_curve,
-    energy_decay_curve_analytic,
+
+from .parametric import (
     air_attenuation_coefficient,
 )
-from .dsp import (
-    find_impulse_response_maximum,
-    find_impulse_response_start,
-    time_shift,
-    preprocess_rir,
-    estimate_noise_energy,
-)
-from .edc import (
-    schroeder_integration,
-    energy_decay_curve_chu,
-    energy_decay_curve_chu_lundeby,
-    energy_decay_curve_lundeby,
-    energy_decay_curve_truncation,
-    intersection_time_lundeby,
-)
+
+from . import edc
+from . import rap
+from . import dsp
+from . import analytic
+from . import parametric
 
 __all__ = [
-    'reverberation_time_linear_regression',
-    'reverberation_time_energy_decay_curve',
-    'schroeder_integration',
-    'energy_decay_curve_analytic',
-    'air_attenuation_coefficient',
-    'find_impulse_response_maximum',
-    'find_impulse_response_start',
-    'time_shift',
-    'preprocess_rir',
-    'energy_decay_curve_chu',
-    'energy_decay_curve_chu_lundeby',
-    'energy_decay_curve_lundeby',
-    'energy_decay_curve_truncation',
-    'estimate_noise_energy',
-    'intersection_time_lundeby',
+    'edc',
+    'rap',
+    'dsp',
+    'analytic',
+    'parametric',
+    'air_attenuation_coefficient'
```

Comment:
```suggestion
```
same here, all pyfar

Reply:
See comment above, will remove once it's part of a new pyfar release.